docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -316,33 +370,60 @@ Now add the following code to your `src/main.ts`:
 24 await shutdown();
 ```
 
-Now when you run
+This local blockchain also provides pre-funded accounts. These lines create local test accounts with test MINA to use for this tutorial:
 
 ```sh
```

Comment:
after https://github.com/o1-labs/docs2/issues/374 is merged, we can use `ignore`